examples/rust/syncing/esplora/src/main.rs

```diff
@@ -16,7 +16,7 @@ const INTERNAL_DESCRIPTOR: &str = "tr(tprv8ZgxMBicQKsPdrjwWCyXqqJ4YqcyG4DmKtjjsR
 
 fn main() -> Result<(), Error> {
     let mut wallet: Wallet = Wallet::create(EXTERNAL_DESCRIPTOR, INTERNAL_DESCRIPTOR)
-        .network(Network::Signet)
+        .network(Network::Testnet)
```

Comment:
We will keep using Signet for the examples if possible (if we used testnet it would be testnet 4 in any case).

But for this PR let's keep using Signet. You can hit the `https://blockstream.info/signet/api/` esplora endpoint for this example.